packages/assert/src/index.ts

```diff
@@ -4,8 +4,10 @@ import * as _ from 'lodash';
 import * as validUrl from 'valid-url';
 
 const HEX_REGEX = /^0x[0-9A-F]*$/i;
+const schemaValidator = new SchemaValidator();
 
 export const assert = {
+    schemaValidator,
```

Comment:
I really don't like this pattern. The reason why is because if a user of this package forgets to add the schemas he later tries to check for by calling `doesConformToSchema`, it will fail to do so.

Reply:
I'd prefer we convert `assert` into a class and explicitly have it instantiated with all the schemas passed into it. If a user calls `doesConformToSchema` with a schema that was not passed in during instantiation, an error is thrown. Does this make sense?

Reply:
This is also what will happen now. It will just throw an error, that that exact schema is not found.

Reply:
Let's leave the code better off then how we found it. Any reservations with making this a class?